README.md

```diff
@@ -50,6 +43,17 @@ configuration, you can run:
 sh scripts/add-chain.sh frontier
 ```
 
+#### Standard chains
+A chain may meet the definition of a **standard** chain. Adding a standard chain is a two-step process.
+
+First, the chain should be added as a frontier chain but with `--standard-chain-candidate` flag set:
+
+```shell
+sh scripts/add-chain.sh standard --standard-chain-candidate
```

Comment:
```suggestion
sh scripts/add-chain.sh frontier --standard-chain-candidate
```

Reply:
@sbvegan Do we need to disambiguate from "standard track" if that is a pre-existing term?

Reply:
@geoknee should we throw an error to avoid having both "standard" and "standard candidate" set (I don't think that makes sense?)